sbk-gem/javadoc/io/gem/params/impl/SbkGemParameters.html

```
@@ -105,15 +104,15 @@ <h1 title="Class SbkGemParameters" class="title">Class SbkGemParameters</h1>
 
 <p>Extends <code>SbkDriversParameters</code> to include SBK driver/logger help, and adds GEM-specific
 options for remote orchestration (nodes, SSH creds/port, SBK and Java deployment,
-copy/delete/deleteafter,
+automatic runtime deployment/cleanup,
 local SBM host/port/idle sleep). Populates typed getters and constructs <a href="../../api/ConnectionConfig.html" title="class in io.gem.api"><code>ConnectionConfig</code></a>
 instances for each target node.
 
 <p>Supported options (help text shows defaults from <a href="../../config/GemConfig.html" title="class in io.gem.config"><code>GemConfig</code></a>):
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p></p> ], start tag match failed [ <p> ] on line 111.

(tag-pair)